src/main/scala/cromwell/binding/RuntimeAttributes.scala

```diff
@@ -0,0 +1,87 @@
+package cromwell.binding
+
+import com.google.api.services.genomics.model.Disk
+import cromwell.binding.AstTools.{AstNodeName, EnhancedAstNode}
+import cromwell.binding.RuntimeAttributes.{Defaults, LocalDisk}
+import cromwell.parser.MemorySize
+import cromwell.parser.WdlParser.{Ast, AstList}
+
+import scala.collection.JavaConverters._
+import scala.language.postfixOps
+
+object RuntimeAttributes {
+  /** Convenience class to convert to Google `Disk` class. */
+  private case class LocalDisk(name: String, sizeGb: Long, diskType: String) {
+    def toDisk: Disk = new Disk().setSizeGb(sizeGb).setType(diskType).setName(name)
+  }
+
+  /** Fallback values if values for these keys are not specified in a task "runtime" stanza. */
+  object Defaults {
+    val Cpu = 1
+    val Disk = Seq(LocalDisk("local-disk", 100L, "LOCAL_SSD").toDisk)
+    val FailOnStderr = false
+    val MemoryInBytes = MemorySize.GB.toBytes(2)
+    val Preemptible = false
+    val Zones = Array("us-central1-a")
+  }
+}
+
+/**
+ * Parses configuration values per WDL/JES specs.
+ */
+case class RuntimeAttributes(ast: Ast) {
+
+  type RuntimeAttributes = Map[String, String]
```

Comment:
I don't really see the need for the type aliases at this level.

Also I'd advocate for not having a big closure as part of the constructor but rather a function which just get called as `val attributes = foo()`